mne/viz/epochs.py

```diff
@@ -69,6 +70,10 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         Figure instance to draw the image to. Figure must contain two axes for
         drawing the single trials and evoked responses. If None a new figure is
         created. Defaults to None.
+    show_times : array-like, shape (n_epochs,) | None
```

Comment:
-1 `time_lags` and `plot_times` `overlay_time`
they have no intuitive meaning to me.
`onset_times`